### DISCUSSION TITLE
When qrz.com auto-fill works?

### DISCUSSION THREAD

**Discussion by ea5wa** [magicbug/Cloudlog#1057] [Q&A]
I don't really know how it works, because sometimes work after fill in the callsign, but sometimes doesn't work....

Thanks

73 de EA5WA Juan Carlos

**Comment by magicbug** [comment]
Could you explain a bit more? Do you mean QRZ rather than LoTW?

**Reply by ea5wa** [reply]
Yes, is not LoTW is qrz.com.... Sorry

**Reply by magicbug** [reply]
Should just be a matter of typing a callsign in and tabbing out of the field and if QRZ returns anything it will populate.

**Comment by dg9vh** [comment]
Sometimes qrz.com has problems to reply... not often but as I saw in the
past this also happens... maybe there was again such a "high
load"-situation...

73 Kim

**Comment by ea5wa** [comment]
In the picture attached you can see it. After typing the callsign, no info coming from qrz is shown

![image](https://user-images.githubusercontent.com/54686689/120888689-fef61200-c5f9-11eb-81a0-546a6b635fda.png)

**Comment by dg9vh** [comment]
[image: image.png]
After callsign and TAB on my side it's working as expected... are you
XML-subscriber? Is your qrz.com account configured correctly?
Maybe if no subscriber, you run into daily limit of queries!?

73 de Kim
DG9VH

Am Sa., 5. Juni 2021 um 12:32 Uhr schrieb ea5wa Juan Carlos <
***@***.***>:

> In the picture attached you can see it. After typing the callsign, no info
> coming from qrz is shown
>
> [image: image]
> <https://user-images.githubusercontent.com/54686689/120888689-fef61200-c5f9-11eb-81a0-546a6b635fda.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1057#discussioncomment-828738>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OXH7UXR6TFT7MTJKE3TRH4KLANCNFSM46BA77LQ>
> .
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen

**Reply by ea5wa** [reply]
I'm not a subscriber. Maybe that's the reason... I don't know, but trying the same with another log software (KLog) is working fine.
That's why I think is a Cloudlog problem

**Reply by myzinsky** [reply]
I think in cloud log this query works only for XML subscribers. Everything else would bump the business model of qrz.com ;-)
